docs/ugcstor-csi.md

```diff
@@ -5,50 +5,60 @@ sidebar_label: cStor
 ---
 ------
 <br>
-
+ 
 <img src="/docs/assets/svg/3-config-sequence.svg" alt="OpenEBS configuration flow" style="width:100%">
-
+ 
 <br>
 <br>
-
-This user guide section provides the operations needed to be performed by the User and the Admin for configuring cStor related tasks. 
-
-   :::note 
-   The recommended approach to provision cStor Pools is to use CStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="https://docs.openebs.io/v260/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
-   :::
-
+ 
+This user guide section provides the operations needed to be performed by the User and the Admin for configuring cStor related tasks.
+ 
+  :::note
+  The recommended approach to provision cStor Pools is to use CStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="https://docs.openebs.io/v260/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
```

Comment:
```suggestion
  The recommended approach to provision cStor Pools is to use CStorPoolCluster(CSPC), the detailed steps have been provided in this document. However, OpenEBS also supports provisioning of cStor Pools using StoragePoolClaim (SPC). For detailed instructions, refer to the <a href="/docs/next/ugcstor.html" target="_blank">cStor User guide(SPC)</a>.<br>
```

Reply:
@kmova, We need to refer to older versions as when this doc will be pushed the SPC content will be entirely replaced with CSPC..Hence we have pointed to  OpenEBS doc version 2.6.0.

Reply:
got it. I think we should move the older doc under the "deprecated" section.